hw/ip/prim/dv/prim_ram_scr/cpp/scramble_model.h

```diff
@@ -40,13 +40,16 @@ std::vector<uint8_t> scramble_addr(const std::vector<uint8_t> &addr_in,
  * @param repeat_keystream Repeat the keystream of one single PRINCE instance if
  *                         set to true. Otherwise multiple PRINCE instances are
  *                         used.
+ * @param repeat_keystream Whether the S&P layer should be used for data
+ * diffusion. In HW this is disabled by default since it interacts adversely
+ * with the end-to-end integrity scheme.
```

Comment:
I think the name is wrong here? I'd probably also drop the "whether" bit, so something like this?
```
 * @param use_sp_layer Use the S&P layer for data diffusion ...
```

Also (nit): Indentation for the following lines.

Reply:
Ah yes, copy paste error...